bin/webpack

```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
-cd client && yarn run build:production
-cd client && yarn run build:test
+if [[ $RAILS_ENV = "production" ]] ; then
+  cd ../client && yarn run build:production
+fi
+
+if [[ $RAILS_ENV = "test" ]] ; then
+  cd ../client && yarn run build:test
+fi
```

Comment:
and we need the else for others, like development

and the ENV should be be NODE_ENV falling back to RAILS

look at the default config/webpack/production.js for webpacker installs